web_tree_many2one_clickable/__manifest__.py

```diff
@@ -2,16 +2,18 @@
 # Copyright 2013 Therp BV (<http://therp.nl>).
 # Copyright 2015 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
 # Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# Copyright 2017 Sodexis <dev@sodexis.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Clickable many2one fields for tree views",
     "summary": "Open the linked resource when clicking on their name",
-    "version": "9.0.1.0.0",
+    "version": "10.0.1.0.0",
     "category": "Hidden",
     "website": "https://odoo-community.org/",
     "author": "Therp BV, "
               "Tecnativa, "
+              "Sodexis, "
```

Comment:
@Yajo, I repeat, it's not the case (see diff), and this is not contribution, it's authorship.